### PR TITLE
Generic cluster usage cleanups

### PR DESCRIPTION
**Pull request by steffenheyne**
fix some issues after merging

### CODE REVIEW

File: workflows/DNA-mapping/DNA-mapping [hunk 1]
```diff
@@ -317,9 +315,9 @@ def main():
     ## create local temp dir and add this path to environment as $TMPDIR variable
     ## on SLURM: $TMPDIR is set, created and removed by SlurmEasy on cluster node
     temp_path = cf.make_temp_dir(args.tempdir, args.outdir, args.verbose)
-    snakemake_exports = ("export TMPDIR="+temp_path+" && ").split()
+    snakemake_exports += ("export TMPDIR="+temp_path+" && ").split()
```
Comment (dpryan79) [inline workflows/DNA-mapping/DNA-mapping]: FYI, this will break if the path contains a space.

Comment (dpryan79) [reply workflows/DNA-mapping/DNA-mapping]: It's fine for now, but in the future we might use something from `shlex` instead.

File: workflows/RNA-seq/RNA-seq [hunk 2]
```diff
@@ -305,11 +305,7 @@ def main():
         cluster_config = cf.merge_dicts(cluster_config, user_cluster_config) # 2) merge/override variables from user_config.yaml
     cf.write_configfile(os.path.join(args.outdir,'cluster_config.yaml'),cluster_config)
 
-    if not args.local:
-        snakemake_module_load += cluster_config["cluster_module_load"]
-    snakemake_module_load = "module load slurm "
-
-    snakemake_module_load = (snakemake_module_load + " && ").split()
+    snakemake_exports = "export PATH=/data/processing/conda/bin:$PATH && ".split()
```
[the remark this answers is not in the document]
Comment (steffenheyne) [reply workflows/RNA-seq/RNA-seq]: you mean we remove this here?

[the remark this answers is not in the document]
Comment (dpryan79) [reply workflows/RNA-seq/RNA-seq]: Yes

[the remark this answers is not in the document]
Comment (steffenheyne) [reply workflows/RNA-seq/RNA-seq]: so until we have a conda package with snakemake as dependency we have to do this manually on the cmdline

[the remark this answers is not in the document]
Comment (dpryan79) [reply workflows/RNA-seq/RNA-seq]: Correct

**Pull request merged**